dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml

```diff
@@ -34,6 +28,12 @@
 
 %h3
   Stay involved with CS Education
+%p
+  = link_to('Six different studies show', 'https://medium.com/@codeorg/cs-helps-students-outperform-in-school-college-and-workplace-66dd64a69536') + ":"
+  children who study computer science perform better in other subjects,
+  excel at problem-solving,
+  and are 17% more likely to attend college.
+  
```

Comment:
Note: Trailing space on this line will cause a lint error on staging. (We should figure out how to get Drone to catch this.)

Reply:
I'm sorry, what trailing space are you referring to?

Reply:
With whitespace rendering enabled in VSCode:

![image](https://user-images.githubusercontent.com/1615761/84187358-c7847200-aa46-11ea-954a-737922af994d.png)

I'm referring to the two space characters on line 36.  See line 28 for what this should look like.

Most IDEs have options both to make whitespace visible like this, and to automatically strip trailing whitespace on save so you don't have to think about it.

Reply:
Ah I see. Thanks!